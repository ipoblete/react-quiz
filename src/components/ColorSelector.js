import React from 'react';
import PropTypes from 'prop-types';

function ColorSelector({ color, background }) {
  return (
    <>
      <label>Font Color:</label>
      <input type="color" name="color" value={color}/>
      <label>Background Color:</label>
      <input type="color" name="background" value={background}/>
    </>
  );
}

ColorSelector.propTypes = {
  color: PropTypes.string.isRequired,
  background: PropTypes.string.isRequired
};

export default ColorSelector;
