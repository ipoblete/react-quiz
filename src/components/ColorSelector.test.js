import React from 'react';
import renderer from 'react-test-renderer';
import ColorSelector from './ColorSelector';

describe('ColorSelector', () => {
  it('matches a snapshot', () => {
    const color = renderer.create(
      <ColorSelector />
    ).toJSON();
    expect(color).toMatchSnapshot();
  });
});
