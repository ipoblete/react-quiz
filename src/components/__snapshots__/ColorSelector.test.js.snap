// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorSelector matches a snapshot 1`] = `
Array [
  <label>
    Font Color:
  </label>,
  <input
    name="fontColor"
    type="color"
  />,
  <label>
    Background Color:
  </label>,
  <input
    name="backgroundColor"
    type="color"
  />,
]
`;
